static/sidebar/js/modal/index.js

```diff
@@ -54,14 +54,19 @@
       prodNode.href = productLink;
       prodNode.innerHTML = '<div class="tvp-product-image" '+productImgStyle+'><div/>';
       thumbsFrag.appendChild(prodNode);
+      
+      var trimmedTitle = product.title.length > 50 ? product.title.substring(0, 50 - 3) + "..." :product.title;
```

Comment:
What's the logic behind:

`substring(0, 50 - 3)`

Why do we need to substract 3? why not doing

`substring(0, 47)`

Can we handle the character qty limit as an option?